cirq-google/cirq_google/workflow/quantum_executable_test.py

```diff
@@ -117,3 +122,51 @@ def test_quantum_executable_inputs():
         )
     with pytest.raises(TypeError):
         _ = QuantumExecutable(spec={'name': 'main'}, circuit=circuit, measurement=measurement)
+
+
+def _get_quantum_executables():
+    qubits = cirq.LineQubit.range(10)
+    return [
+        QuantumExecutable(
+            spec=ExampleSpec(name=f'example-program-{i}'),
+            circuit=_get_random_circuit(qubits, random_state=i),
+            measurement=BitstringsMeasurement(n_repetitions=10),
+        )
+        for i in range(3)
+    ]
+
+
+def test_quantum_executable_group_to_tuple():
+    exes1 = list(_get_quantum_executables())
+    exes2 = tuple(_get_quantum_executables())
+
+    eg1 = QuantumExecutableGroup(exes1)
+    eg2 = QuantumExecutableGroup(exes2)
+    assert hash(eg1) == hash(eg2)
+    assert eg1 == eg2
+
+
+def test_quantum_executable_group_container(tmpdir):
```

Comment:
Nit: Can you split this out so \_\_len__, \_\_iter__, \_\_repr__ and \_\_str__ are tested in one test and the json serialization deserialization consistency is tested in another ?

Reply:
I can and I have

Reply:
😎